### DISCUSSION TITLE
Create sub tasks

### DISCUSSION THREAD

**Discussion by PEPERSO** [minbrowser/min#1517] [Ideas]
Hello ! I've been using Min tasks a lot these days because I need to have lots of tabs openned, and keep them for next session. But it would be easier to keep things separated if I could set subtasks, like this task and subtask organization:

1. Find a flat
1.1 On one renting website
1.2 On another
1.3 etc.
2. Read news
3. Follow courses
3.1 Project 1
3.2 Info help
3.3 etc.

With only tasks it is much harder to keep a clean organization like this. I know that the purpose of Min is not to have 70 tabs openned, but I need to find a flat and to follow my courses, and I can't lose those tabs, so...

**Comment by Syndamia** [comment]
You can also use bookmarks, because they have tags, so you can have a bookmark for "Find a flat" and then another for "On one renting website", etc.

**Comment by PEPERSO** [comment]
Damn, I didn't thought of this type of bookmark use, but yes you're right ! Anyway, subtasks could be interesting too, I think
